src/services/Functional/Utilities.ts

```diff
@@ -5,3 +5,55 @@ export function isScreenPortrait() {
   if (screen.availHeight > 0.8 * screen.availWidth) return true;
   return false;
 }
+
+/**
+ * An identifier to hold the current animation frame request.
+ * useful when it is needed to cancel a particular animation frame
+ */
+export let animationFrameRequest: number = 0;
+
+/**
+ * Animates confetti gun for a certain amount of time
+ * @param {Object} confettiHandler - Handler which will draw the confetti on the canvas
+ * @param {Number} duration - How long the confetti should be animated for
+ * @param {Array} colors - Colors for the confetti
+ */
+export function throwConfetti(
+  confettiHandler: any,
```

Comment:
why will the type of confettiHandler be `any`?

Reply:
![image](https://user-images.githubusercontent.com/73693628/164176280-d9bf5a4d-d590-4f47-b5c0-513312020e41.png)